go/api/openapi.json

```diff
@@ -50,13 +50,7 @@
           }
         },
         "type": "object",
-        "required": [
-          "requestId",
-          "detail",
-          "status",
-          "title",
-          "type"
-        ]
+        "required": ["requestId", "detail", "status", "title", "type"]
```

Comment:
@perkinsjr not sure why `pnpm fmt` had to format this. Would've expected this to be caught by CI upstream.

Keep or leave?

Reply:
keep it as is, that's on me